### PR TITLE
Add WILC token support

### PR DESCRIPTION
**Pull request by fahimaltinordu**
Dear General Bytes Team,

This PR is to add Wrapped ILCOIN (WILC) extension to BATM to support WILC in the atm.

Please review the PR and suggest us any changes if required.

Best regards

### CODE REVIEW

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import java.util.Set;
+
+public class CoinmarketinfoRateSource implements IRateSource {
+    private static final Logger log = LoggerFactory.getLogger(CoinmarketinfoRateSource.class);
+
+    private final ICoinmarketinfoAPI api;
+    private String preferredFiatCurrency = FiatCurrency.USD.getCode();
+
+    public CoinmarketinfoRateSource(String preferedFiatCurrency) {
+        api = RestProxyFactory.createProxy(ICoinmarketinfoAPI.class, "https://api.coinmarketinfo.com:9445");
```
Comment (generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java:42]: 9945? Please use url that uses port 443 (https)

**Review by generalbytes — CHANGES_REQUESTED** [review]

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java (hunk 1 above)
Comment (fahimaltinordu) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java:42]: Actually it is in https but under a different port, if it is problem we can change the source from coinmarketinfo to nomics.com

Comment (fahimaltinordu) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java]: Okey, we replaced coinmarketinfo rate source with nomics @generalbytes.
Please let me know if there is anything missing still.
Best regards

Comment (generalbytes) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ilcoin/sources/coinmarketinfo/CoinmarketinfoRateSource.java]: Thank you.

**Pull request merged**